exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -199,6 +199,9 @@ def send_bytes(bytes, timeout:) # rubocop:disable Metrics/AbcSize, Metrics/Cyclo
           rescue Net::OpenTimeout, Net::ReadTimeout
             retry if backoff?(retry_count: retry_count += 1, reason: 'timeout')
             return FAILURE
+          rescue OpenSSL::SSL::SSLError
+            retry if backoff?(retry_count: retry_count += 1, reason: 'openssl_error')
```

Comment:
This is out of the scope of this PR but I would be interested in knowing more details of the exception if I ran into this in production. Anyone have an appetite for me to open an issue about adding more exception details to the log?

Reply:
I had wondered about that, too, with [the logging done for encoding errors](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/exporter/otlp/test/opentelemetry/exporter/otlp/exporter_test.rb#L192-L209).